src/services/swap/providers/lightning-labs/loop-service.ts

```diff
@@ -0,0 +1,191 @@
+// https://lightning.engineering/loopapi/index.html#service-swapclient
+import util from "util"
+
+import * as grpc from "@grpc/grpc-js"
+import { getSwapConfig } from "@config"
+import { SwapClientNotResponding, SwapServiceError } from "@domain/swap/errors"
+
+import { SwapState as SwapStateType } from "@domain/swap/index"
+
+import { SwapType as DomainSwapType } from "@domain/swap"
+
+import { SwapClientClient } from "./protos/loop_grpc_pb"
```

Comment:
@nicolasburtey 

Is there a way to exclude these generated proto files from complying with the "no implicit any rules"?

Reply:
@dolcalmi may know more. I think you need to generate the associated definition file: 

ie: [src/services/dealer-price/proto/services/price/v1/price_service_grpc_pb.d.ts](https://github.com/GaloyMoney/galoy/blob/main/src/services/dealer-price/proto/services/price/v1/price_service_grpc_pb.d.ts)